packages/analytics/shippers/elastic_v3/server/src/server_shipper.ts

```diff
@@ -216,40 +215,29 @@ export class ElasticV3ServerShipper implements IShipper {
   }
 
   private setInternalSubscriber() {
-    // Check the status of the queues every 1 second.
+    // Create an emitter that emits when 10s have passed since the last time we sent the data
+    const atLeast10secondsSinceLastSent$ = interval(SECOND).pipe(
```

Comment:
I appreciate the very explicit name here 😅 But perhaps it should be something more general, like `MIN_TIME_SINCE_LAST_SEND`, so that if we need to further adjust the window later on, we only have to update it its value in one place?